kissnet.hpp

```diff
@@ -925,9 +918,7 @@ namespace kissnet
 #ifdef KISSNET_USE_OPENSSL
 			else if constexpr (sock_proto == protocol::tcp_ssl) //only TCP is a connected protocol
 			{
-				memcpy(&socket_output, getaddrinfo_results->ai_addr, sizeof(SOCKADDR));
-
-				int error = syscall_connect(sock, reinterpret_cast<SOCKADDR*>(&socket_output), sizeof(SOCKADDR));
+				int error = syscall_connect(sock, reinterpret_cast<SOCKADDR*>(getaddrinfo_results->ai_addr), socklen_t(getaddrinfo_results->ai_addrlen));
```

Comment:
The reinterpret_cast id not needed here.

Reply:
Done.